fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1293,4 +1293,12 @@
      */
     @ConfField
     public static String http_api_extra_base_path = "";
+
+    /**
+     * Whether to support the creation of alpha rowset tables.
+     * The default is false and should only be used in emergency situations,
+     * this config should be remove in some future version
+     */
+    @ConfField
```

Comment:
mutable = true?

Reply:
This configuration is just a transitional configuration, the purpose is to deal with some unexpected special scenarios and facilitate debugging and reproducing problems. Later versions will completely ban segment v1, so there is no need to make easy-to-use settings for this configuration.